src/ontology/mondo-edit.obo

```diff
@@ -542679,6 +542690,35 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-5208-3432
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7717" xsd:anyURI
 
+[Term]
+id: MONDO:0100573
+name: combined generalized and focal epilepsy
+def: "Any epilepsy where patients have both generalized and focal seizure types, with interictal and/or ictal EEG findings that accompany both seizure types. Patients with Dravet syndrome and Lennox-Gastaut syndrome may have combined focal and generalized epilepsy." [https://www.epilepsydiagnosis.org/epilepsy/focal-generalized-epilepsy-groupoverview.html]
+is_a: MONDO:0005027 {source="https://www.epilepsydiagnosis.org/epilepsy/focal-generalized-epilepsy-groupoverview.html"} ! epilepsy
+intersection_of: MONDO:0005384 ! focal epilepsy
+intersection_of: MONDO:0100574 ! generalized epilepsy
```

Comment:
I think the logical definition is fine. However, it means that the classification is such that this term is a child instead of a sibling of "focal epilepsy" and "generalized epilepsy" (which seems weird). 
Let's discuss if this is what we want.

Reply:
I think that's expected?

@matentzn could you take a quick look and verify?
I added a new term 'combined generalized and focal epilepsy' with the logical def:
'focal epilepsy'
 and 'generalized epilepsy'

and this is the resulting hierarchy:

<img width="438" alt="image" src="https://github.com/user-attachments/assets/79811d2a-ca0d-4c45-96f4-484aa6cfd864">

thx!

Reply:
@nicolevasilevsky I think the classification is correct if we use that logical definition. 
But I think this classification will not make sense to the experts (ie they probably expect to see this term as a sibling class).